docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -71,51 +72,96 @@ $ tsh ssh --cluster=leaf host
 $ tsh clusters
 ```
 
-Leaf clusters also have their own restrictions on user access, i.e.
-*permissions mapping* takes place.
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+```code
+# Log in using the root "root" cluster credentials:
+$ tsh login --proxy=mytenant.teleport.sh
+
+# SSH into some host inside the "root" cluster:
+$ tsh ssh host
+
+# SSH into the host located in another cluster called "leaf"
+# The connection is established through root.example.com:
+$ tsh ssh --cluster=leaf host
+
+# See what other clusters are available
+$ tsh clusters
+```
 
-**Once a connection has been established it's easy to switch from the "root" root cluster**
+</TabItem>
+</Tabs>
+
+Once a connection has been established, it's easy to switch from the root cluster.
 ![Teleport Cluster Page](../../../img/trusted-clusters/teleport-trusted-cluster.png)
 
 Let's take a look at how a connection is established between the "root" cluster
 and the "leaf" cluster:
 
 ![Tunnels](../../../img/tunnel.svg)
 
-This setup works as follows:
+This setup works as follows. The "leaf" creates an outbound reverse SSH tunnel
```

Comment:
Per https://english.stackexchange.com/a/394875 , I think you still need a colon after "as follows".